source/stylesheets/components/_case_study.scss

```diff
@@ -0,0 +1,125 @@
+// CaseStudy
+//
+// A compoent to showcase a project
+//
+// .CaseStudy--articles - Uses oncostats' background color
+// .CaseStudy--oncostats - Users articles' background color
+// .CaseStudy--left - Puts the image on the left
+//
+// markup:
+// <div class="CaseStudy {$modifiers}">
+//   <img class="CaseStudy-image">
+//   <div class="CaseStudy-details">
+//     <div class="CaseStudy-fixedBackground"></div>
+//     <h2 class="CaseStudy-title">Some title</h2>
+//     <p>Some html</p>
+//     <p>More HTML</p>
+//   </div>
+// </div>
+//
+// Styleguide 5.22
+
+$CaseStudy-colors: (articles, #cdb187), (oncostats, #6dd4e0);
+$CaseStudy-shadow: 0 20px 60px 0 rgba(#000, 0.25);
+
+.u-overflowHidden {
+  overflow: hidden;
+}
+
+.CaseStudy {
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+
+  max-width: $Theme-grid-maxWidth;
+  margin: 0 auto;
+
+  @include media('>smallDesktop') {
```

Comment:
With the increasing ubiquity of the web and the increasing number of different device sizes, shapes and types, would it make sense to stop using these aliases and use only the pixel value? The reasoning being that the style of `smallDesktop` might actually be show on a large tablet, or small tv, or something else, which is not inherently wrong, but kind of defeats the abstraction on which the alias was based in the first place.

I know this is very meta and I don't think this is a blocker to the merge of the PR, but it is something I have been wondering and felt this would be as good a place as any to start the conversation.

/cc @subvisual/css